java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -268,24 +270,22 @@ private boolean learnedConstraintForDomains(LearnedConstraint lc) {
       return flowBuilder.build();
     }
 
-    private List<JavaFileScannerContext.Location> flowFromYields(ExplodedGraph.Edge edge) {
-      ExplodedGraph.Node parent = edge.parent;
-      List<Integer> argumentIndices = correspondingArgumentIndices(symbolicValues, parent);
+    private Set<List<JavaFileScannerContext.Location>> flowFromYields(ExplodedGraph.Edge edge) {
+      Set<MethodYield> methodYields = edge.yields();
+      if (methodYields.isEmpty()) {
+        return ImmutableSet.of(ImmutableList.of());
```

Comment:
Empty set would made `addEdge` more complicated, because no yield would need to be handled as a special case. Like this we can treat no yield and yield with no flow the same way